ruby.c

```diff
@@ -280,6 +280,23 @@ show_usage_line(const struct ruby_opt_message *m,
     }
 }
 
+#if USE_YJIT
+void
+rb_show_usage_line(const char *name, const char *description,
```

Comment:
```suggestion
rb_print_usage_line(const char *name, const char *description,
```

Reply:
I want the functions names in ruby.c to be consistent with each other. It uses "show" in many places, so I changed the other name to be consistent with it instead https://github.com/ruby/ruby/pull/8583/commits/2132f3793f0a6ca63e286f4cdf7fc3e89caf9237.